pkg/asset/installconfig/aws/validation_test.go

```diff
@@ -354,42 +358,55 @@ func TestValidate(t *testing.T) {
 		publicSubnets:  validPublicSubnets(),
 		exptectErr:     `^platform\.aws\.serviceEndpoints: Invalid value: (.|\n)*: \[failed to find endpoint for service "ec2": (.|\n)*, failed to find endpoint for service "elasticloadbalancing": (.|\n)*, failed to find endpoint for service "iam": (.|\n)*, failed to find endpoint for service "route53": (.|\n)*, failed to find endpoint for service "s3": (.|\n)*, failed to find endpoint for service "sts": (.|\n)*, failed to find endpoint for service "tagging": (.|\n)*\]$`,
 	}, {
-		name: "custom region invalid service endpoints some provided",
+		name: "custom region valid service endpoints",
 		installConfig: func() *types.InstallConfig {
 			c := validInstallConfig()
 			c.Platform.AWS.Region = "test-region"
 			c.Platform.AWS.AMIID = "dummy-id"
-			c.Platform.AWS.ServiceEndpoints = validServiceEndpoints()[:3]
+			c.Platform.AWS.ServiceEndpoints = validServiceEndpoints()
 			return c
 		}(),
 		availZones:     validAvailZones(),
 		privateSubnets: validPrivateSubnets(),
 		publicSubnets:  validPublicSubnets(),
-		exptectErr:     `^platform\.aws\.serviceEndpoints: Invalid value: (.|\n)*: \[failed to find endpoint for service "elasticloadbalancing": (.|\n)*, failed to find endpoint for service "route53": (.|\n)*, failed to find endpoint for service "sts": (.|\n)*, failed to find endpoint for service "tagging": (.|\n)*$`,
 	}, {
-		name: "custom region valid service endpoints",
+		name: "AMI not provided for unknown region",
```

Comment:
why this change? hm...

Reply:
This was leftover cruft from a recent rebase. Fixed now.